tests/libs/datasets/timeseries_test.py

```diff
@@ -92,15 +92,15 @@ def test_multi_region_to_from_timeseries_and_latest_values(tmp_path: pathlib.Pat
         )
     )
     region_97111 = multiregion.get_one_region(Region.from_fips("97111"))
-    assert region_97111.date_indexed.at["2020-04-02", "m1"] == 2
+    assert region_97111.date_indexed.at[pd.to_datetime("2020-04-02"), "m1"] == 2
```

Comment:
Not entirely sure why, but accessing dates with strings no longer worked. This was happening before the change above to add `.freq("D)` to `date_indexed`, so I don't believe it was caused by that